docs/_process.md

```diff
@@ -1,9 +0,0 @@
-## Design and Build Process
```

Comment:
Not used as far as I can tell.

Reply:
I think there is a place for a document like this in the future, but for now, I've let that short text live in the "Overview" as well.

Reply:
This was created to capture some of what James and I talked about during our design sprint. I don't think there's anything wrong with leaving it as breadcrumbs and changing "Sketch.Cloud" to "Design in Figma" as a way to help give context to new folks and providing us a place to start as we continue to flesh this out.

Reply:
Addressed, added a "Process" section to the overview page.